src/tests/ftest/util/soak_test_base.py

```diff
@@ -431,8 +431,8 @@ def job_completion(self, job_id_list):
                         if time.time() + int(180) < self.end_time:
                             failed_harasser_msg = self.launch_harasser(
                                 offline_harasser, self.pool)
-                            # wait 2 minutes to issue next harasser
-                            time.sleep(120)
+                            # wait 5 minutes to issue next harasser
+                            time.sleep(300)
```

Comment:
Does this make sense in the context of `if time.time() + int(180) < self.end_time`?